### PR TITLE
Rollout tfbridge v3.81.0

### PR DESCRIPTION
**Pull request by iwahbe**
Rebased on top of https://github.com/pulumi/pulumi-newrelic/pull/713.

Upstream's schema description fields are non-deterministic, so we introduce a patch to make them deterministic. 

The nondeterminism was exposed by correctly finding schema data (introduced in https://github.com/pulumi/pulumi-newrelic/pull/713).

I'll open an issue in this repo when this PR merges to track the patch. I'll also open an issue upstream to see if we can upstream the fix.

### CODE REVIEW

File: sdk/dotnet/Cloud/AwsGovcloudLinkAccount.cs [hunk 1]
```diff
@@ -24,7 +24,6 @@ namespace Pulumi.NewRelic.Cloud
     /// 
     /// ## Example Usage
     /// 
-    /// &lt;!--Start PulumiCodeChooser --&gt;
```
Comment (VenelinMartinov) [inline sdk/dotnet/Cloud/AwsGovcloudLinkAccount.cs:27]: Are these dropped intentionally?

File: sdk/dotnet/Inputs/ServiceLevelObjectiveTimeWindowRollingArgs.cs [hunk 2]
```diff
@@ -12,15 +12,9 @@ namespace Pulumi.NewRelic.Inputs
 
     public sealed class ServiceLevelObjectiveTimeWindowRollingArgs : global::Pulumi.ResourceArgs
     {
-        /// <summary>
```
Comment (VenelinMartinov) [inline sdk/dotnet/Inputs/ServiceLevelObjectiveTimeWindowRollingArgs.cs:15]: Some more missed descriptions - why is this changing?

File: sdk/dotnet/Cloud/AwsGovcloudLinkAccount.cs (hunk 1 above)
Comment (iwahbe) [reply sdk/dotnet/Cloud/AwsGovcloudLinkAccount.cs:27]: Yes. That was https://github.com/pulumi/pulumi-terraform-bridge/pull/1887.

File: sdk/dotnet/Inputs/ServiceLevelObjectiveTimeWindowRollingArgs.cs (hunk 2 above)
Comment (iwahbe) [reply sdk/dotnet/Inputs/ServiceLevelObjectiveTimeWindowRollingArgs.cs:15]: While I can't speak to this specific doc, I'm assuming upstream doesn't have a label for each branch in their unrolled recursive structure, and our refined docs parsing is correctly identifying that upstream doesn't have a description here.

**Review by VenelinMartinov — APPROVED** [review]

**Pull request merged**